pyfar/utils.py

```diff
@@ -207,8 +212,10 @@ def concatenate_channels(signals, caxis=0, broadcasting=False):
         return pf.Signal(data, signals[0].sampling_rate,
                          n_samples=signals[0].n_samples,
                          domain=signals[0].domain,
```

Comment:
This is correct but maybe a little implicit: The domain will always be `time` because we querried s.time in line 209. How about being more explicit

```suggestion
                         domain='time',
```